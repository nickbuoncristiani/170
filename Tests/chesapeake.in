39
25
1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38 39
18 33 2 27 4 26 38 32 11 8 6 37 14 29 10 25 9 34 12 7 31 16 24 19 3
20
x 1.18103 1.52029 1.03133 1.04659 1.88435 x x x 1.41416 x 1.55254 1.86199 1.49708 x x x x x x x x 1.88632 x x x x x x x x x x x x x x x x
1.18103 x 1.14032 1.37354 1.64825 1.78937 1.32348 1.78071 1.3709 1.38223 1.58409 1.53957 x x x x x x x x x x x x x x x x x x x x x x x x x x x
1.52029 1.14032 x x x x 1.41987 1.04935 x 1.00775 x 1.72374 1.90465 1.29063 x x x x x x x x x x 1.17469 1.31426 1.3554 1.93575 1.05516 x x x x x x x x x x
1.03133 1.37354 x x x x x x x 1.10631 x 1.08529 1.21349 x 1.45844 x x x x x x x x x x x x x x 1.80223 x x x x x x x x x
1.04659 1.64825 x x x x x x x 1.62698 x 1.4027 1.98547 x 1.0466 x x x x x x x x x x x x x 1.37324 1.00695 1.24782 x x x x x x x x
1.88435 1.78937 x x x x x x x 1.53186 x 1.2594 1.35107 x 1.16882 x x x x x x x x x x x x x 1.26491 x x x x x x x x x x
x 1.32348 1.41987 x x x x x x 1.21266 x 1.42603 1.65618 x 1.68698 x x x x x x x x x x x x x 1.26398 x x x 1.02705 1.96336 x 1.61059 1.22093 1.01671 1.79266
x 1.78071 1.04935 x x x x x x 1.17073 x 1.50141 1.1481 x 1.15238 x x x x x x x x x x x x x 1.6775 x x x x x x 1.8069 1.93261 1.87326 x
x 1.3709 x x x x x x x x 1.70128 x x x x x x x x x x x x 1.50205 x x x x x x x x x x x x x x x
1.41416 1.38223 1.00775 1.10631 1.62698 1.53186 1.21266 1.17073 x x 1.57465 x 1.19415 1.58939 1.03291 x x x x x x x 1.16943 1.82259 1.46252 x x x x x x x x x x x x x x
x 1.58409 x x x x x x 1.70128 1.57465 x x x x x x x x x x x 1.90753 x x x x x x x x x x x x x x x x x
1.55254 1.53957 1.72374 1.08529 1.4027 1.2594 1.42603 1.50141 x x x x 1.05116 1.10059 x 1.45009 1.4493 1.30799 1.13634 1.44127 1.65755 1.49647 1.5223 1.06691 1.27917 1.98676 1.18998 1.56802 x 1.38006 1.46286 1.22896 1.01186 1.69359 1.75088 1.1011 1.17738 1.61835 1.58629
1.86199 x 1.90465 1.21349 1.98547 1.35107 1.65618 1.1481 x 1.19415 x 1.05116 x 1.17091 1.56497 x x x x x x x x x x x x x x x x x x x x x x x x
1.49708 x 1.29063 x x x x x x 1.58939 x 1.10059 1.17091 x 1.34438 x x x x x x x x x 1.60612 x x x x x x x x x x x x x x
x x x 1.45844 1.0466 1.16882 1.68698 1.15238 x 1.03291 x x 1.56497 1.34438 x 1.71999 1.41994 1.09546 1.86152 1.89929 1.20307 1.76232 x x 1.90407 1.83342 1.63314 1.79974 x 1.8202 1.96357 1.44092 1.99138 1.68387 1.9725 1.25812 1.80389 1.23455 1.22939
x x x x x x x x x x x 1.45009 x x 1.71999 x 1.41781 x x x x x x x x x x x x x 1.52983 1.35015 1.02925 1.29859 1.5028 x x x x
x x x x x x x x x x x 1.4493 x x 1.41994 1.41781 x 1.61422 1.58213 1.33048 1.18751 x x x x x x x x x x x x x x x x x x
x x x x x x x x x x x 1.30799 x x 1.09546 x 1.61422 x x x x x x x x x x x x 1.11943 1.93474 1.3792 1.16711 1.42304 1.36259 x x x x
x x x x x x x x x x x 1.13634 x x 1.86152 x 1.58213 x x x x x x x x x x x x 1.5744 x x 1.40376 x x x x x x
x x x x x x x x x x x 1.44127 x x 1.89929 x 1.33048 x x x x 1.90791 x x x x x x x x x x x x x x x x x
x x x x x x x x x x x 1.65755 x x 1.20307 x 1.18751 x x x x x x x x x x x x 1.2948 1.60716 1.40617 1.87466 x 1.00786 1.84439 x x x
x x x x x x x x x x 1.90753 1.49647 x x 1.76232 x x x x 1.90791 x x x x x x x x x x x x x x x x x x x
1.88632 x x x x x x x x 1.16943 x 1.5223 x x x x x x x x x x x 1.00915 x x x x x x x x x x x x x x x
x x x x x x x x 1.50205 1.82259 x 1.06691 x x x x x x x x x x 1.00915 x x x x x x x x x x x x x x x x
x x 1.17469 x x x x x x 1.46252 x 1.27917 x 1.60612 1.90407 x x x x x x x x x x x x x x x x x x x x x x x x
x x 1.31426 x x x x x x x x 1.98676 x x 1.83342 x x x x x x x x x x x x x 1.9816 x x x x x x x x x x
x x 1.3554 x x x x x x x x 1.18998 x x 1.63314 x x x x x x x x x x x x x 1.54037 x x x x x x x 1.26712 x x
x x 1.93575 x x x x x x x x 1.56802 x x 1.79974 x x x x x x x x x x x x x 1.00015 x x x x x x x x x x
x x 1.05516 x 1.37324 1.26491 1.26398 1.6775 x x x x x x x x x x x x x x x x x 1.9816 1.54037 1.00015 x 1.6173 1.48036 1.18929 1.14628 1.47333 1.47938 1.33843 1.39339 1.44027 1.54763
x x x 1.80223 1.00695 x x x x x x 1.38006 x x 1.8202 x x 1.11943 1.5744 x 1.2948 x x x x x x x 1.6173 x x x x x x x 1.68634 x x
x x x x 1.24782 x x x x x x 1.46286 x x 1.96357 1.52983 x 1.93474 x x 1.60716 x x x x x x x 1.48036 x x x x x x x x x x
x x x x x x x x x x x 1.22896 x x 1.44092 1.35015 x 1.3792 x x 1.40617 x x x x x x x 1.18929 x x x x x x x x x x
x x x x x x 1.02705 x x x x 1.01186 x x 1.99138 1.02925 x 1.16711 1.40376 x 1.87466 x x x x x x x 1.14628 x x x x x x x x 1.65979 x
x x x x x x 1.96336 x x x x 1.69359 x x 1.68387 1.29859 x 1.42304 x x x x x x x x x x 1.47333 x x x x x x x x x x
x x x x x x x x x x x 1.75088 x x 1.9725 1.5028 x 1.36259 x x 1.00786 x x x x x x x 1.47938 x x x x x x x x x x
x x x x x x 1.61059 1.8069 x x x 1.1011 x x 1.25812 x x x x x 1.84439 x x x x x x x 1.33843 x x x x x x x x x 1.16241
x x x x x x 1.22093 1.93261 x x x 1.17738 x x 1.80389 x x x x x x x x x x x 1.26712 x 1.39339 1.68634 x x x x x x x x x
x x x x x x 1.01671 1.87326 x x x 1.61835 x x 1.23455 x x x x x x x x x x x x x 1.44027 x x x 1.65979 x x x x x x
x x x x x x 1.79266 x x x x 1.58629 x x 1.22939 x x x x x x x x x x x x x 1.54763 x x x x x x 1.16241 x x x
