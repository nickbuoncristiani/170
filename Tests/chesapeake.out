20 12 31 16 34 7 3 29 37 12 32 18 12 38 8 10 24 12 26 3 2 11 9 2 12 20
18
12 12
31 31
16 16
34 34
7 7
3 3 14
29 29
37 37
12 33 27 6
32 32
18 18
12 19
38 38
8 8
10 25 10 4
24 24
26 26
2 2
11 11
9 9
